articles/sentinel/customer-managed-keys.md

```diff
@@ -19,18 +19,32 @@ ms.author: yelevin
 
 > [!NOTE]
-> - The Azure Sentinel CMK capability is provided only to **new customers**.
+> - The Azure Sentinel CMK capability is provided only to **new workspaces**.
 >
-> - Access to this capability is controlled by Azure feature registration. You can request access by contacting azuresentinelCMK@microsoft.com. Pending requests will be approved according to the available capacity.
+> - Onboarding a CMK workspace to Sentinel is supported only via REST API and not via the Azure Portal. Azure Resource Manager (ARM) templates are currently not supported for CMK onboarding.
 >
-> - The CMK capability is only available to customers sending 1TB per day or more. You will receive information about additional pricing when you apply to Microsoft to provision CMK on your Azure subscription. Learn more about [Log Analytics pricing](../azure-monitor/logs/manage-cost-storage.md#log-analytics-dedicated-clusters).
+> - Once a workspace is onboarded as a CMK workspace, setting it as a non-CMK workspace by de-linking it from the dedicated Log Analytics CMK cluster, or setting the dedicated Log Analytics cluster as non-CMK is not supported and might lead to undefined and undesired behavior.
+>
+> - Once a workspace was onboarded to Azure Sentinel as a non-CMK workspace, linking it to a CMK Log Analytics cluster, or setting the Log Analytic cluster its currently linked to, as a CMK cluster, is not supported, and will lead to its Azure Sentinel data being encrypted by a Microsoft Managed Key instead of the Customer Managed Key.
+>
+> - Azure Sentinel supports System Assigned Identities in CMK configuration. Therefore, the dedicated Log Analytics cluster's identity should be of System Assigned type. It is recommended that it will be the identity automatically assigned to the Log Analytics cluster upon its creation.
+>
+> - Changing the Customer Managed Key to another key (with another Uri) is currently not supported. Changing the key should be done by rotating it.
+>
+> - Please consult Azure Sentinel's Product Group about any CMK changes to a production workspace or a Log Analytics cluster at azuresentinelCMK@microsoft.com.
+>
+> - Upon completing the steps in this guide, and prior to using the workspace, please contact azuresentinelCMK@microsoft.com for onboarding confirmation.
+>
+> - The CMK capability requires a dedicated Log Analytics cluster with 1TB a day or more of reserved capacity. Several workspaces can be linked to the same dedicated cluster, and they will share the same customer managed key.
+>
+> - You will receive information about additional pricing when you apply to Microsoft to provision CMK on your Azure subscription. Learn more about [Log Analytics dedicated cluster](../azure-monitor/logs/manage-cost-storage.md#log-analytics-dedicated-clusters).
```

Comment:
"Apply to" Microsoft? Or contact Microsoft as described in the previous step?

Reply:
when the customer creates the dedicated cluster in LogA

Reply:
I changes it and included a full path to the logA pricing for dedicated clusters